run_e3sm.template.sh

```diff
@@ -18,7 +15,7 @@ main() {
 # Machine and project
 readonly MACHINE=pm-cpu
 # BEFORE RUNNING:  CHANGE this to your project
-readonly PROJECT="e3sm"
+readonly PROJECT="$(sacctmgr show user $USER format=DefaultAccount | tail -n1 | tr -d ' ')"
```

Comment:
How portable is that sacctmgr command?

Reply:
I was chatting with Naser about this. It's true we can't assume a system has slurm.  And I was thinking more about this in general.  We don't want to add things that we don't like to these scripts -- having settings like `PROJECT=e3sm` or `PROJECT=cli115` is generally exactly what we want as we share with others.  But if we could have a test (say external in python) that could tell us a user is *not* in these projects (again up to us how exactly to implement), then maybe we could not even set PROJECT and let system us default project.

Reply:
Need to improve the comment. Instead of "# BEFORE RUNNING:  CHANGE this to your project"  use
"#NOTE:  The command below will return your default project on SLURM-based systems. If you are not using SLURM or need a different project, remove the command and set it directly"